components/KASToolBar/kaspathedit.pas

```diff
@@ -367,6 +368,26 @@ procedure TKASPathEdit.CreateWnd;
   FAutoComplete:= not SHAutoCompleteX(Handle, FObjectTypes);
 end;
 
+procedure TKASPathEdit.setTextAndSelect( newText:String );
```

Comment:
This variant is not needed, second variant should be without `IFDEF`. It will be used if native Windows autocomplete failed.